azurerm/helpers/azure/app_service.go

```diff
@@ -328,11 +329,9 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 				"java_version": {
 					Type:     schema.TypeString,
 					Optional: true,
-					ValidateFunc: validation.StringInSlice([]string{
-						"1.7",
-						"1.8",
-						"11",
-					}, false),
+					ValidateFunc: validation.StringMatch(
+						regexp.MustCompile(`^(1\.7|1\.8|11)`),
+						`Invalid Java version provided`),
```

Comment:
since the App Settings is shared between the App Service and App Service Slot resource; could we also document this in the App Service Slot resource too?